#! /bin/sh -e
## 06_disable_rpath.dpatch by Martin Loschwitz <madkiss@madkiss.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Disables the rpath settings in qmake.conf

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
    -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1;;
esac

exit 0

diff -ruN qt-x11-free-3.1.1.old/mkspecs/linux-g++/qmake.conf qt-x11-free-3.1.1/mkspecs/linux-g++/qmake.conf
--- qt-x11-free-3.1.1.old/mkspecs/linux-g++/qmake.conf	2002-12-09 10:41:32.000000000 +0100
+++ qt-x11-free-3.1.1/mkspecs/linux-g++/qmake.conf	2003-02-18 19:07:18.000000000 +0100
@@ -53,7 +53,7 @@
 QMAKE_LFLAGS_PLUGIN	= $$QMAKE_LFLAGS_SHLIB
 QMAKE_LFLAGS_SONAME	= -Wl,-soname,
 QMAKE_LFLAGS_THREAD	=
-QMAKE_RPATH		= -Wl,-rpath,
+QMAKE_RPATH		=
 
 QMAKE_LIBS		=
 QMAKE_LIBS_DYNLOAD	= -ldl
